consul/README.md

```diff
@@ -1,32 +1,143 @@
 # Consul Integration
 
-## Overview
+# Overview
 
-Get metrics from consul service in real time to:
+The Datadog Agent collects many metrics from Consul nodes, including those for:
 
-* Visualize and monitor consul states
-* Be notified about consul failovers and events.
+* Total Consul peers
+* Service health - for a given service, how many of its nodes are up, passing, warning, critical?
+* Node health - for a given node, how many of its services are up, passing, warning, critical?
+* Network coordinates - inter- and intra-datacenter latencies
 
-## Installation
+The _Consul_ Agent can provide further metrics via DogStatsD. These metrics are more related to the internal health of Consul itself, not to services which depend on Consul. There are metrics for:
```

Comment:
EDIT: ok, some marketing pages *do* talk a little about the how, e.g.:

"Submit custom metrics from your Node applications using our http API or the Singly node bindings. See them in Datadog for graphing, slice-and-dice, and correlation."

But here again, "graphing, slice-and-dice, and correlation" applies to almost any integration. So I guess we want to include these generic, pitchy phrases in all Overviews, so long as there's sufficient variety of phrases across the integrations?

One other general question: what content length should we aim for? e.g. 2-3 sentences minimum, 5 max?